### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by aspala**
Hello!

I fixed some typos. :)

Signed-off-by: Andreas Solleder <sollederandreas@gmail.com>

### CODE REVIEW

File: guides/v1.10.0/testing/testing-xhr.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ If you're using a promise, but there's a chance it might resolves after the test
 
 ## AJAX
 
-AJAX requests are the most prevelant use case where you will be creating promises.  While testing it's likely you will want to mock your AJAX requests to the server.  Below we've included examples for [ic-ajax](https://github.com/instructure/ic-ajax). Feel free to use other mocking libraries such as [Mockjax](https://github.com/appendto/jquery-mockjax), but it's important to note, that Mockjax and other libraries are unaware of the run loop and won't wrap their resolve in a run call.  This may resolve in promises being run outside the realm of the run loop and will result in errors.
+AJAX requests are the most prevalent use case where you will be creating promises.  While testing it's likely you will want to mock your AJAX requests to the server.  Below we've included examples for [ic-ajax](https://github.com/instructure/ic-ajax). Feel free to use other mocking libraries such as [Mockjax](https://github.com/appendto/jquery-mockjax), but it's important to note, that Mockjax and other libraries are unaware of the run loop and won't wrap their resolve in a run call.  This may resolve in promises being run outside the realm of the run loop and will result in errors.
```
Comment (aspala) [inline guides/v1.10.0/testing/testing-xhr.md:182]: The change here is `prevelant` -> `prevalent`

File: guides/v1.11.0/testing/testing-xhr.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ If you're using a promise, but there's a chance it might resolves after the test
 
 ## AJAX
 
-AJAX requests are the most prevelant use case where you will be creating promises.  While testing it's likely you will want to mock your AJAX requests to the server.  Below we've included examples for [ic-ajax](https://github.com/instructure/ic-ajax). Feel free to use other mocking libraries such as [Mockjax](https://github.com/appendto/jquery-mockjax), but it's important to note, that Mockjax and other libraries are unaware of the run loop and won't wrap their resolve in a run call.  This may resolve in promises being run outside the realm of the run loop and will result in errors.
+AJAX requests are the most prevalent use case where you will be creating promises.  While testing it's likely you will want to mock your AJAX requests to the server.  Below we've included examples for [ic-ajax](https://github.com/instructure/ic-ajax). Feel free to use other mocking libraries such as [Mockjax](https://github.com/appendto/jquery-mockjax), but it's important to note, that Mockjax and other libraries are unaware of the run loop and won't wrap their resolve in a run call.  This may resolve in promises being run outside the realm of the run loop and will result in errors.
```
Comment (aspala) [inline guides/v1.11.0/testing/testing-xhr.md:189]: The change here is `prevelant` -> `prevalent`

File: guides/v2.13.0/controllers/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ## Controllers
 
-Controllers behave like a specialized type of Component that is rendered by
+Controllers behave like a specialised type of Component that is rendered by
```
Comment (ijlee2) [inline guides/v2.13.0/controllers/index.md:3]: This one, let's keep it spelled with a "z". I think the guides follow the American spelling in the case of s/z variation.

```suggestion
Controllers behave like a specialized type of Component that is rendered by
```

**Comment by ijlee2** [comment]
For organization, I grouped the typos that @aspala found by the affected Markdown file:

### 1. components/block-content.md

What changed: occured → occurred

- [ ] v3.15.0
- [x] v3.16.0
- [x] v3.17.0
- [x] v3.18.0
- [x] release


### 2. configuring-ember/index.md

What changed: enviroment → environment

- [x] v2.0.0
- [x] v2.1.0


### 3. contributing/repositories.md (my extra request)

What should change: seperate → separate

- [ ] v1.10.0
- [ ] v1.11.0


### 4. cookbook/user_interface_and_interaction/specifying_data_driven...md

What changed: specifiy → specify

- [ ] v1.10.0
- [x] v1.11.0


### 5. models/customizing-serializers.md

What changed: recieved → received

- [x] v2.0.0
- [x] v2.1.0


### 6. templates/displaying-a-list-of-items.md

What changed: space-seperated → space-separated

- [x] v2.0.0
- [x] v2.1.0


### 7. testing/testing-helpers.md

- [x] 2.13.0


### 8. testing/testing-xhr.md

What changed: implicitely → implicitly, prevelant → prevalent

- [x] v1.10.0
- [x] v1.11.0


### 9. tutorial/deploying.md

What changed: enviroment → environment

- [x] v2.3.0
- [x] v2.4.0
- [x] v2.5.0
- [x] v2.6.0
- [x] v2.7.0
- [x] v2.8.0
- [x] v2.9.0
- [x] v2.10.0


### 10. wip/rails.md

What changed: appropirate → appropriate

- [x] v1.10.0

**Comment by aspala** [comment]
Thank you for your feedback!

As you suggested, I changed the commit-history to better reflect the changes for each typo.

: )

**Pull request merged**